### PR TITLE
Adds cube bom implementation.

### PR DESCRIPTION
**Pull request by dipak-pawar**
#### Short description of what this resolves:
Adds bom for Arquillian Cube

**Fixes**: #571

### CODE REVIEW

**Comment by bartoszmajsak** [comment]
Good idea @dipak-pawar. Thanks! Since you are fiddling with `pom.xml`, can you also exclude all the `*-ftest` modules from releasing?

It simply boils down to putting 

```xml
  <plugin>
    <groupId>org.apache.maven.plugins</groupId>
    <artifactId>maven-deploy-plugin</artifactId>
    <configuration>
        <skip>true</skip>
    </configuration>
  </plugin>
```

in every relevant module.

File: docs/bom.adoc [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+----
+<dependency>
+    <groupId>org.arquillian.cube</groupId>
+    <artifactId>arquillian-cube-requirement</artifactId>
+    <scope>test</scope>
+    <type>pom</type>
+</dependency>
+----
+
+==== Arquillina Cube docker
```
Comment (bartoszmajsak) [inline docs/bom.adoc]: Minor thing - let's keep headers capitalized

Comment (bartoszmajsak) [inline docs/bom.adoc]: And crazy idea - how about autogenerating these sections based on the content of POM? :)

Comment (dipak-pawar) [inline docs/bom.adoc]: Idea looks good. Need to look how can we implement it?

Comment (bartoszmajsak) [inline docs/bom.adoc]: Sure, take your time - it's something of a great added value, but nothing super important for the moment.

**Comment by bartoszmajsak** [comment]
LGTM @lordofthejars - anything missing from your POV?

**Comment by lordofthejars** [comment]
Love it.

**Pull request merged**